Re-added view_metabolic_model_details that apparently didn't make the…

briehl opened the pull request:
… last cut.

There may be others that come up later.

nlharris commented on methods/view_metabolic_model_details/display.yaml:
```diff
@@ -0,0 +1,47 @@
+#
+name     : View Metabolic Model Details
+subtitle : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```
should say "Bring up a detailed view of a metabolic model". (Not sure how to edit in the PR.)

nlharris commented on methods/view_metabolic_model_details/display.yaml:
```diff
@@ -0,0 +1,47 @@
+subtitle : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
+tooltip  : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```
Same here.

nlharris commented on methods/view_metabolic_model_details/display.yaml:
```diff
@@ -0,0 +1,47 @@
+
+
+description : |
+    Bring up a detailed view of your metabolic model within the narrative. [7]
```
should say "Bring up a detailed view of a metabolic model".

briehl commented:
Edited and changed, and removed the [7] and [7.1] numbers, too.
That method won't show up in the methods panel, anyway, but it's needed for the drag and drop viewer to work.
